### PR TITLE
NEW Article: Migrate to Office 365 FAQ

### PR DESCRIPTION
**Pull request by DonNateR**
From PR #2140

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-414f967ca5/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-56e00ff6c0/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-ad5e30125e/how-to/)

**Comment by kmsholcomb** [comment]
Hi @DonNateR. See the attached files for my edits. Please let me know if you have any questions. Thanks!

[Rackspace Email and Microsoft Exchange upgrade to Office 365 FAQ_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1136438/Rackspace.Email.and.Microsoft.Exchange.upgrade.to.Office.365.FAQ_editsKH.pdf)
[Rackspace Email and Microsoft Exchange upgrade to Office 365 FAQ_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1136437/Rackspace.Email.and.Microsoft.Exchange.upgrade.to.Office.365.FAQ_editsKH.docx)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-67356a8395/how-to/)

File: content/office-365/upgrade-rackspace-email-and-microsoft-exchange-upgrade-to-office-365-faq.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Yes, all users must create new profiles. We’ve created an online mail client setup tool that walks users through configuring their email for use on all of the most popular desktop and mobile platforms: [https://emailhelp.rackspace.com/](https://emailhelp.rackspace.com/)
+
+#### Am I charged for both environments during this transitional period?
+
+Yes. Your invoice will reflect fees for both environments while they remain active on your account. We can work with you to determine a timeline for migration that accommodates any billing concerns. After your transition to Office 365 is complete, you must delete your Rackspace Email and Microsoft Exchange mailboxes and request that those licenses be removed from your account to avoid future billing.
+
+#### Is there be downtime during the migration?
+
+No. Admins are responsible for updating their domain's DNS records. The migration specialist provides you with a list of records that must be updated. Rackspace does not have access to your externally hosted DNS and cannot accept access to external DNS hosting accounts.
```
Comment (catlook) [inline content/office-365/upgrade-rackspace-email-and-microsoft-exchange-upgrade-to-office-365-faq.md]: The "Admin's are responsible" section is repeated in the question below, and doesn't seem to apply to the question of whether or not there is downtime.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-02bf71d3d5/how-to/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-83c2c96e37/how-to/)

**Pull request merged**